docs/template-format.md

```diff
@@ -5,6 +5,7 @@ The following variables are available in the templates used when formatting note
 | Variable      | Type     | Description                                                              |
 |---------------|----------|--------------------------------------------------------------------------|
 | `path`        | string   | File path to the note, relative to the current directory                 |
+| `full-path`   | string   | File path to the note, full path including the notebook directory        |
```

Comment:
Sounds useful 👍 

Could you rename it to `abs-path` instead? That's what I've been using in a similar context: https://github.com/mickael-menu/zk/blob/4d893b9bd6934170d181b5dafe460802700bec1c/docs/tool-fzf.md#template-context